webpack/webpack.common.js

```diff
@@ -57,6 +58,34 @@ module.exports = {
             title: 'Message-sender-ui',
             template: path.resolve(appSrc, 'index.html'),
             favicon: false,
-        })
-    ],   
+        }),
+        new CopyPlugin({
```

Comment:
move this plugin to production config, delete it from dev also.
and delete initEditor.js empty file